src/SFA.DAS.EAS.Portal.UnitTests/Worker/EventHandlers/Paye/WhenAddedPayeSchemeEventReceived.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
```

Comment:
The test and filename could be renamed to the name of the event hander + `Tests`, to fit in with the existing test names.

Reply:
Done